pytest.ini

```diff
@@ -0,0 +1,19 @@
+[pytest]
```

Comment:
I don't think this file is actually needed, markers should work automatically.

Reply:
You are right, but it gets rid of warning messages in the end of the test run.